### DISCUSSION TITLE
message d'erreur lors de la tentative d'installation de Klipper TMC Autotune

### DISCUSSION THREAD

**Discussion by gigis57** [Guilouz/Klipper-Flsun-Speeder-Pad#129] [Q&A]
Bonjour,

J'ai un message d'erreur lors de la tentative d'installation de Klipper TMC Autotune.

Je suis entièrment à jour et j'ai 3 instances : 

Voici le message d'erreur:  [ERROR] Klipper service not found, please install Klipper first!

Y a til une solution ? Merci

**Comment by kaktuss77** [comment]
hello, si c'est pour la première instance ou tu as la V400, il faut télécharger le script install.sh, le modifier (avec nano) et remplacer 

`if [ "$(sudo systemctl list-units --full -all -t service --no-legend | grep -F 'klipper.service')" ]; then`

par 

`if [ "$(sudo systemctl list-units --full -all -t service --no-legend | grep -F 'klipper-1.service')" ]; then`

car si tu as une instance le service s'appelle bien klipper.service, si plusieurs alors klipper-1.service , klipper-2service et klipper-3.service

![image](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/76732050/1cc13021-082f-4fdc-909a-6721cc54544d)


PS : Je n'ai pas (encore) tester
